static/app/utils/withIssueTags.spec.tsx

```diff
@@ -89,7 +89,7 @@ describe('withIssueTags HoC', function () {
 
     expect(
       screen.getByText(
-        /assigned: me, my_teams, \[me, my_teams, none\], #best-team-na, foo@example.com, joe@example.com/
```

Comment:
Very small nit, but it would satisfy my OCD if the order was: 
* me
* my_teams
*  **none**
*  **[me, my_teams, none]**

The individual options are seen before they're seen again in the array, and their order lines up with the order in the array.